tests/engine/daemon/test_execmanager.py

```diff
@@ -123,10 +124,12 @@ def test_retrieve_files_from_list(
     target = tmp_path_factory.mktemp('target')
 
     create_file_hierarchy(file_hierarchy, source)
+    runner = get_manager().get_runner()
 
     with LocalTransport() as transport:
         node = generate_calcjob_node(workdir=source)
-        execmanager.retrieve_files_from_list(node, transport, target, retrieve_list)
+        runner.loop.run_until_complete(execmanager.retrieve_files_from_list(node, transport, target, retrieve_list))
+        # await execmanager.retrieve_files_from_list(node, transport, target, retrieve_list)
```

Comment:
Some commented out code0that can be removed?

Reply:
done